README.md

```diff
@@ -3,13 +3,37 @@
 - List 3 reasons why asymptotic analysis may be misleading with respect to
   actual performance in practice.
 
+  - asymptotic analysis is used to compare growth rates for algorithms. This means that constant factors and lower order factors are ignored. a function that has a practical runtime of 2n^2 + 5 would have a much better performance on smaller data sets than a fuction with a runtime of 4242n + 7log(n) + 5555, even if their asymptotic complexities would be evaluated to O(n^2) and O(n) respectively.
+ 
+  - asymptotic analysis operates under assumptions about the data set used, ie best case, worst case, average case. In reality data used may adhere to certain patterns, such as always being partially or nearly sorted. algorithms that have good best case but bad average or worst case complexity may be faster on said data sets, while an algorithm with the same best worst and average case may be less effective. Asymptotic analysis is a tool to quantify and compare algorithms based on assumptions that in reality may not be true. Asymptotic complexity gives a heirarchy of speeds, constant time faster than logarithmic time faster than linear time etc. but this heirarchy does not alwady remain consistent when used in real world applications, on data sets like in the examples given. More has to be known about the problem and the algorithm than asymptotic complexity of a given algorithm to determine the fastest option
```

Comment:
This sounds very fuzzy. Can you be more specific here please, ideally with some examples?